README.md

```diff
@@ -25,13 +25,14 @@ In this challenge, create a web API around the following resources: `Projects` a
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] Mention two parts of Express that you learned about this week.
+`MiddleWare & server side routing`
 
 - [ ] Describe Middleware?
-
+    `Is this bodyguard right, that it verifies certain requirements that are need to be met, if requirements are not met, it does something, if the are, it enables the next middleware (if there's any) to be run`
 - [ ] Describe a Resource?
-
+   `I believe is this chunk of data that is rendered or used when a specific end point is requested`
 - [ ] What can the API return to help clients know if a request was successful?
-
+    `A res.status with a friendly message saying that their request was successful`
```

Comment:
all 200 codes are positive codes for status
400s are errors over the network, like missing data or dropped packets etc. 
500s are internal errors like the server is down or theres an issue with the commands being run server side